code/api/src/seeders/1-user.js

```diff
@@ -25,7 +30,11 @@ module.exports = {
         role: params.user.roles.user,
         createdAt: new Date(),
         updatedAt: new Date(),
-        image: 'https://en.pimg.jp/045/948/028/1/45948028.jpg'
+        image: 'https://en.pimg.jp/045/948/028/1/45948028.jpg',
+        address_line1: '5678 Here Ave',
+        city: 'Pueblo',
+        state: 'CO',
+        zipcode: 85623
       }
```

Comment:
This looks great! Smart to include the admin and the user

Reply:
Good call making sure that address line 2 can be blank!